error_reporting/unit_tests/test_client.py

```diff
@@ -128,6 +128,32 @@ def test_report(self):
         self.assertLess(report_location['lineNumber'], 150)
 
 
+class TestFlaskHelper(unittest.TestCase):
+
+    def test_flask_helper(self):
+        from google.cloud.error_reporting import build_flask_context
+        request = _Request('http://google.com', 'GET',
```

Comment:
Empty line between imports and code (I realize we have lots of unit tests violating this, but no need to pile on)